cpp/mg_utility/mg_graph.hpp

```diff
@@ -13,20 +13,20 @@
 
 namespace mg_graph {
 
+enum class GraphType : std::uint8_t { kDirectedGraph, kUndirectedGraph };
```

Comment:
Do you need k before the names if it's part of the enum?
k is used for constants, enum values are something else.

Reply:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names 

Followed this guide.

Reply:
Interesting, nice catch!